sequencing/clinical_annotator/resources/home/dnanexus/reformatHGMD.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env python
+"""Script to reformat the HGMD PRO vcf file for one annotation per line.
+"""
+
+__author__ = "Thomas Nate Person"
+__license__ = "GPLv3"
+__email__ = "thomas.n.person@gmail.com"
+
+import sys
+import gzip
+import collections
+
+
+vcf_file=None
+
+if sys.argv[1].endswith(".gz"):
+	vcf_file = gzip.open(sys.argv[1],'r')
+else:
+	vcf_file = open(sys.argv[1],'r')
+
+source=""
+
+ids = dict()
+
+last_site_id = ""
+hgmd_annotations=collections.OrderedDict()
+
+new_vcf=collections.OrderedDict()
+
+for line in vcf_file:
+	if line.startswith("#"):
```

Comment:
Again recommend using pyvcf/cyvcf. Otherwise, parse the header in a separate for loop

```
for line in vcf_file:
  if line.startswith('#'):
    ...
  else:
    break
```

Then create a csv.reader and parse the variant rows